portals/publisher/source/src/app/components/Apis/Details/Environments/Environments.jsx

```diff
@@ -1058,13 +1059,23 @@ export default function Environments() {
         >
             <Grid item className={classes.shapeRec} />
             <Grid item className={clsx(classes.shapeCircaleBack, classes.shapeCircle)}>
-                <Grid
-                    onClick={handleClickOpen}
-                    className={clsx(classes.shapeDottedStart, classes.shapeCircle)}
-                    style={{ cursor: 'pointer' }}
-                >
-                    <AddIcon style={{ fontSize: 30 }} className={classes.plusIconStyle} />
-                </Grid>
+                {api.advertiseInfo && api.advertiseInfo.advertised ? (
+                    <Grid
+                        className={clsx(classes.shapeDottedStart, classes.shapeCircle)}
+                        style={{ cursor: 'pointer' }}
+                    >
+                        <AddIcon color='disabled' style={{ fontSize: 30 }} className={classes.plusIconStyle} />
```

Comment:
add style prop to the class

Reply:
Done with https://github.com/wso2/apim-apps/pull/130/commits/21fcbb82f68bc91cc952b7c05f7acfd2ef40532a